src/components/Dragons/Dragons.jsx

```diff
@@ -0,0 +1,40 @@
+// React hooks
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+// Thunk Action creators
+import { fetchDragons } from '../../redux/dragons/dragons';
```

Comment:
- [ ] Nice work done on your Dragons component, kindly check this import `../../redux/dragons/dragons`, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a LowerCase `r`edux

Reply:
Thank @danielufeli for mentioning that ❤